nixos/modules/system/boot/systemd.nix

```diff
@@ -652,6 +652,7 @@ in
 
     services.logind.lidSwitchExternalPower = mkOption {
       default = config.services.logind.lidSwitch;
+      defaultText = "config.services.logind.lidSwitch";
```

Comment:
Should probably drop the `config.` prefix, since the `config.` prefix isn't really user-facing. To the user, it is just `services.logind.lidSwitch` which does the business.

Reply:
Done, and also updated the other occurrence.